docs/src/api/class-browsercontext.md

```diff
@@ -216,6 +216,8 @@ The default browser context cannot be closed.
 :::
 
 ## async method: BrowserContext.cookies
+* langs:
+  - alias-csharp: GetCookiesAsync
```

Comment:
I do, for most use cases (https://github.com/microsoft/playwright/blob/b7125097693c587598741f79a7e2e703b492f6c7/utils/doclint/generateDotnetApi.js#L399) but this one is special, because it also accepts an argument. 

I'd much rather have this alias'd than hard-code an edge case in the generator.